services/brig/test/integration/API/User/Auth.hs

```diff
@@ -540,11 +558,18 @@ getCookieId c = maybe (error "no cookie value")
                       (fromByteString (cookie_value c))
 
 listCookies :: HasCallStack => Brig -> UserId -> Http [Auth.Cookie ()]
-listCookies b u = do
-    rs <- get (b . path "/cookies" . header "Z-User" (toByteString' u)) <!!
+listCookies b u = listCookiesWithLabel b u []
+
+listCookiesWithLabel :: HasCallStack => Brig -> UserId -> [CookieLabel] -> Http [Auth.Cookie ()]
+listCookiesWithLabel b u l = do
+    rs <- get (b . path "/cookies"
+                 . queryItem "labels" labels
```

Comment:
Does everything still work if `labels` is empty?

Reply:
Yup, that's the [default](https://github.com/wireapp/wire-server/blob/develop/services/brig/src/Brig/User/Auth/Cookie.hs#L138)